README.md

```diff
@@ -22,9 +44,9 @@ In `./my-projects/foodadvisor/api`, run the following from your command line:
 yarn && yarn run seed && yarn develop
 ```
 
-Then, follow [Step 2. Create an admin user](https://strapi.io/documentation/v3.x/getting-started/quick-start.html#_2-create-an-administrator-user) to create an `Administrator`.
+Then, follow [the instructions here](https://strapi.io/documentation/v3.x/getting-started/quick-start.html#_2-create-an-administrator-user) to create an **Administrator**.
```

Comment:
Need to change link: https://strapi.io/documentation/developer-docs/latest/getting-started/quick-start.html#_2-create-an-administrator-user